aggregate/component-site/pom.xml

```diff
@@ -20,7 +20,7 @@
     <jbosstools-site>http://download.jboss.org/jbosstools/updates/stable/luna/</jbosstools-site>
     <jbosstools-tests-site>http://download.jboss.org/jbosstools/updates/nightly/coretests/master/</jbosstools-tests-site>
     <update.site.name>JBoss Tools - ${name}</update.site.name>
-    <snapshotLocation>components</snapshotLocation>
+    <snapshotLocation>component</snapshotLocation>
```

Comment:
Not necessary. Just makes this consistent with the other places the variable is used, relative to the folder name. Eg, coretests-site/pom.xml  has snapshotLocation = coretests.